src/AWS.Deploy.CLI/Commands/CommandFactory.cs

```diff
@@ -307,6 +311,80 @@ private Command BuildListCommand()
             return listCommand;
         }
 
+        /// <summary>
+        /// Builds the top level command called "deployment-project" which supports the creation and saving on the
+        /// CDK deployment project.
+        /// </summary>
+        /// <returns>An instance of the <see cref="Command"/> class</returns>
+        private Command BuildDeploymentProjectCommand()
+        {
+            var deploymentProjectCommand = new Command("deployment-project",
+                "Save the CDK deployment project inside a user provided directory path.");
```

Comment:
Because this is more then just a CDK project lets change "CDK deployment project" to just "deployment project". I don't want users to think they can just run the cdk CLI on this project.

Reply:
Fixed